suzieq/engines/pandas/interfaces.py

```diff
@@ -270,7 +271,7 @@ def _check_ipaddr(x, fld1, fld2, reason):
                     .fillna({i: [] for i in if_df.index})
 
         if 'vlanList' not in if_df.columns:
-            if_df['vlanList'] = []
+            if_df['vlanList'] = [[] for i in repeat(None, len(if_df))]
```

Comment:
```suggestion
            if_df['vlanList'] = list(repeat([], len(if_df)))
```
If we use itertools something like this should be enough.
Or without using any extension:
```suggestion
            if_df['vlanList'] = [[] for i in range(len(if_df))]
```

Reply:
Ack